### PR TITLE
 chapter 9: Advance login

### PR DESCRIPTION
**Pull request by nguyenquangphuong97**

### CODE REVIEW

**Comment by nguyenquangphuong97** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
 
-  # Returns true if the user is logged in, false otherwise.
   def logged_in?
     current_user.present?
   end
 
+  def forget user
+    user.forget
+    cookies.delete(:user_id)
+    cookies.delete(:remember_token)
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: gộp lại đi em
cookies.delete [:user_id, :remember_token]

Comment (nguyenquangphuong97) [reply app/helpers/sessions_helper.rb]: fixed

**Comment by nguyenquangphuong97** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**